src/library_fs.js

```diff
@@ -2022,15 +2002,37 @@ FS.staticInit();` +
       openRequest.onerror = onerror;
     },
 
-    // Allocate memory for an mmap operation. This allocates space of the right
-    // page-aligned size, and clears the padding.
-    mmapAlloc: function(size) {
-      var alignedSize = alignMemory(size, {{{ POSIX_PAGE_SIZE }}});
-      var ptr = _malloc(alignedSize);
-      while (size < alignedSize) HEAP8[ptr + size++] = 0;
-      return ptr;
-    }
-  }
+    // Removed v1 functions
+#if ASSERTIONS
+    absolutePath: function() {
+      abort('FS.absolutePath has been removed');
```

Comment:
How about mentioning the replacement in these error messages? That would also help confirm in review that we are only removing things with a more modern variant.

Reply:
Okay, for some like `createFolder` I can suggest `mkdir`. `absolutePath` was not used, tested, or documented anywhere in the project, so I thought we could just remove it, but considering it's just a one-line wrapper around PATH_FS I can add an explanation for it too. What about for `joinPath`? Still just suggest `PATH.join` even though it does more than just wrap it?

Reply:
Okay, I've done that. People will have to account for the different functionality of the newer functions, but that's what they have to do anytime something is deprecated.